Chelakhovl/HW5/task2.py

```diff
@@ -0,0 +1,9 @@
+n = int(input("Enter a number (n): "))
+
```

Comment:
good idea
I would recommend you to use a more meaningful variable name
and look at this code
n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])


##############

list_of_number = [2, 1, 4, 6, 1, 4, 8, 6]

for i in range(len(list_of_number)):
    list_of_number[i] = float(list_of_number[i])
print(list_of_number)